hapi/error.py

```diff
@@ -9,7 +13,7 @@ def __init__(self):
         self.msg = ''
         self.reason = ''
 
-    def __nonzero__(self):
+    def __bool__(self):
```

Comment:
Missing `__nonzero__ = __bool__`

Reply:
Sorry could you please provide more details? http://python-future.org/compatible_idioms.html#custom-nonzero-vs-bool-method here just renames ```__nonzero__``` to ```__bool__```.

Reply:
https://portingguide.readthedocs.io/en/latest/core-obj-misc.html#customizing-truthiness-bool

Otherwise, the __nonzero__ method is not overriden.